src/components/Wizard/messageTypes.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export type MessageTemplate =
+  | "cfr"
```

Comment:
Looks like the linter is complaining about these double quotes and the semicolon on line 11

Reply:
I filed another PR to disable that linting because we're using prettier and not JS Standard. https://github.com/mozilla/limelight/pull/19

Reply:
Sounds good!